Identity/PermanentIdentity.cs

```diff
@@ -72,7 +72,7 @@ public byte[] UserSecret
             }
         }
 
-        public SecretPermanentIdentity(UserToken userToken, string appId = "")
+        private SecretPermanentIdentity(UserToken userToken, string appId = "")
```

Comment:
What does it have to do with dropping user token class? Split in another commit?